src/System.Threading.Tasks/ref/System.Threading.Tasks.csproj

```diff
@@ -2,12 +2,15 @@
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.0.10.0</AssemblyVersion>
     <OutputType>Library</OutputType>
-    <NuGetTargetMoniker>.NETStandard,Version=v1.3</NuGetTargetMoniker>
+    <NuGetTargetMoniker>.NETStandard,Version=v1.7</NuGetTargetMoniker>
+    <PackageTargetFramework>netstandard1.7;netcore50</PackageTargetFramework>
   </PropertyGroup>
   <ItemGroup>
+    <SuppressPackageTargetFrameworkCompatibility Include="netcore50" />
```

Comment:
Also seemed wierd to me, but this is how @sepidehMS merged her PR initially so I thought there was a reason behind it. @sepidehMS any special reason why we did this ?